### PR TITLE
Mission updates

### PR DESCRIPTION
**Pull request by NotoriousBims**
UK Mission updates - deleted a few seasonal missions that will get generated new when that time comes.

Mission title updates and mostly credit updates. As of this push all UK missions have credits that have been manually checked and adjusted for proper credits.

### CODE REVIEW

File: missions/uk/77.json [hunk 1]
```diff
@@ -1 +0,0 @@
-{"missionId": "77", "missionName": "Patient Transfer", "requirements": [{"requirement": "max. patients", "qty": "1"}], "credits": "0"}
```
Comment (Suchty112) [inline missions/uk/77.json:1]: why delete it? the mission still exists
https://www.missionchief.co.uk/einsaetze.json and search id":77

**Review by Suchty112 — CHANGES_REQUESTED** [review]
why delete the seasonal missions? they can stay in xD
Or is there a reason for that?
Next year they might be generated incorrectly again. Therefore I would leave them in

File: missions/uk/77.json (hunk 1 above)
Comment (jackbayliss) [reply missions/uk/77.json:1]: I would also say to leave it in, if they do change the missions next year then we can just manually fix it I guess.

**Review by jackbayliss — CHANGES_REQUESTED** [review]
I would re-add the seasonal missions, as they will get re-added by the generator bot anyway.

File: missions/uk/77.json (hunk 1 above)
Comment (NotoriousBims) [reply missions/uk/77.json:1]: Yeah that wasn't supposed to get deleted. That one was a mistake.

**Comment by NotoriousBims** [comment]
I deleted the seasonal ones because I can't confirm their credit values right now. I would rather they get re-added by the generator at that time when they come up again so we can confirm and check credit values before they get merged.

File: missions/uk/77.json (hunk 1 above)
Comment (NotoriousBims) [reply missions/uk/77.json:1]: Odd - patient transfer doesn't show up in the games "possible mission" list but it does show in that einsaetze.json linked above.

**Comment by Suchty112** [comment]
If you really wanted to have the credit prices of the seasonal missions, you only had to do a quick search on github :) . 
There is a file from the LSS Manager with all the missions. You can also find the seasonal missions in the json linked above :) . The game has been handing them over for about 3-4 months now. 

to read the einsaetze.json better, you can then simply search for a `beautifier json` on google. then you might find everything better ;)

**Pull request closed without merging**